runtime/src/lib.rs

```diff
@@ -118,10 +118,10 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("kulupu"),
 	impl_name: create_runtime_str!("kulupu"),
 	authoring_version: 5,
-	spec_version: 18,
+	spec_version: 19,
 	impl_version: 0,
 	apis: RUNTIME_API_VERSIONS,
-	transaction_version: 9,
+	transaction_version: 10,
```

Comment:
Don't think this needs a bump

Reply:
Indeed, in this case the underlying encoding does not change. But I kind of want to keep the habit of bumping transaction version when there's interface changes. It won't hurt anyway.